### ISSUE TITLE
Subscript is not being styled 

### ISSUE THREAD

**Issue by HerbCaudill** [keybase/book-content#40]
See the last paragraph under https://book.keybase.io/docs/teams/puk#cryptographic-specifics for example. 

On book site, subscripts are styled as normal text. 

![image](https://user-images.githubusercontent.com/2136620/80584697-5a76c900-8a12-11ea-96b7-9efd0bc24d38.png)

On docs site, subscripts are styled correctly. 

![image](https://user-images.githubusercontent.com/2136620/80584669-5185f780-8a12-11ea-93cc-8e609bbb40ae.png)

**Comment by shiflett** [comment]
Yeah, it looks like the markup is correct, and we need a style defined for `<sub>`:

```
Also, whenever the key rolls over, the previous seed <em>s</em><sub>i</sub> is encrypted with <em>c</em><sub><em>i</em>+1</sub>
```
